packages/use-hooks/src/useBreakpoints/useBreakpoints.ts

```diff
@@ -0,0 +1,41 @@
+import preset from '@scalar/themes/tailwind'
+import { useMediaQuery } from '@vueuse/core'
+import { type Ref, computed, unref } from 'vue'
+
+type Screen = keyof typeof preset.theme.screens
```

Comment:
Can we make the import not load the whole Tailwind preset? That might help with keeping the bundle size increase smaller.

```suggestion
import { theme } from '@scalar/themes/tailwind'
import { useMediaQuery } from '@vueuse/core'
import { type Ref, computed, unref } from 'vue'

type Screen = keyof typeof theme.screens
```

Reply:
It looks like we can't do this because theme isn't actually exported separately 😞